site/content/docs/5.3/examples/features/index.html

```diff
@@ -66,35 +66,35 @@ <h2 class="pb-2 border-bottom">Columns with icons</h2>
     <div class="row g-4 py-5 row-cols-1 row-cols-lg-3">
       <div class="feature col">
         <div class="feature-icon d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-2 mb-3">
-          <svg class="bi" width="1em" height="1em"><use xlink:href="#collection"/></svg>
+          <svg class="bi" width="1em" height="1em" aria-hidden="true"><use xlink:href="#collection"/></svg>
```

Comment:
note that `aria-hidden="true"` is unnecessary if the svg doesn't contain any text elements. SRs will ignore them the same way they ignore a series of purely empty presentational `<div>`s and `<span>`s

Reply:
(just as a heads-up before you laboriously go and add it to *every*, *single*, *svg*)

Reply:
Thanks for your answer :-) Our accessibility experts asked us to add aria-hidden on decorative SVGs in our Boosted doc, so I did it on Bootstrap when relevant. There aren't so much in the doc, but a lot more in the examples...! So if you're ok with this I can stop there and submit the PR?

Reply:
wonder what the rationale provided by the accessibility experts there was.

personally, I have no problem with having all the `aria-hidden`s added, it just felt a bit wasteful/pointless. but *if* we're going to add them, then I'd say it should cover all instances, as it would be more confusing if we had a mix of "some `<svg>`s do have it, others don't". so i'd say either remove the ones you did from this PR, or...go through all the rest